content/docs/command-reference/exp/apply.md

```diff
@@ -0,0 +1,138 @@
+# exp apply
+
+Apply the changes from a previously run experiment into the workspace.
```

Comment:
```suggestion
Apply the changes from a previous experiment to the
<abbr>workspace</abbr>.
```
> "into" does sound better for some reason... but I think the right phrase is "apply to" per https://hinative.com/en-US/questions/1240152

Reply:
Maybe link "experiment" to the index for now (we can link to some concept page or guide later).